content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -211,6 +211,29 @@ RUM supports several propagator types to connect resources with backends that ar
       - `b3multi`: [B3 multiple headers](https://github.com/openzipkin/b3-propagation#multiple-headers) (`X-B3-*`)
 
 {{% /tab %}}
+
+{{% tab "Android RUM" %}}
+1. Set up RUM to connect with APM as described above.
+2. Configure the `OkHttpClient` interceptor with the list of internal, first-party origins and the tracing header type to use as follows:
+    ```java
+    val tracedHosts = mapOf("example.com" to setOf(TracingHeaderType.TRACECONTEXT), 
+                          "example.eu" to setOf(TracingHeaderType.DATADOG))
+
+    val okHttpClient = OkHttpClient.Builder()
+        .addInterceptor(DatadogInterceptor(tracedHosts))
+        .addNetworkInterceptor(TracingInterceptor(tracedHosts))
+        .eventListenerFactory(DatadogEventListener.Factory())
+       .build()
+    ```
+    
+    `TracingHeaderType` in an enum representing the following tracing header types:
+      - `.DATADOG`: Datadog's propagator (`x-datadog-*`)
+      - `.TRACECONTEXT`: [W3C Trace Context](https://www.w3.org/TR/trace-context/) (`traceparent`)
```

Comment:
Can you format the links like in ln 168. For more examples and best practices, see this [guide](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Links)

```suggestion
      - `.TRACECONTEXT`: [W3C Trace Context][1] (`traceparent`)
```
If you want to see a staging preview of your changes, rename your branch to <slack_username>/<branch>. Find your slack username [here](https://dd.slack.com/account/settings#username).

Reply:
Given that lines 209-211 are equivalent to this one but for the browser platform shouldn't those also be changed?

Reply:
Likewise for this PR: https://github.com/DataDog/documentation/pull/16531

Reply:
Yes, they should follow the same convention. If it's easier, feel free to make a separate PR for formatting links.

Reply:
Yeah, then I'd prefer to do a separate PR to unify all 3 platforms.